gen/installer/azure.py

```diff
@@ -150,8 +149,7 @@ def gen_templates(user_args, arm_template):
         results.templates[arm_template + '.json'],
         variant_cloudconfig['master'],
         variant_cloudconfig['slave'],
-        variant_cloudconfig['slave_public']
-        )
+        variant_cloudconfig['slave_public'])
```

Comment:
This change: is it _enforced_ by one of the newly introduced checks? Then I don't want that to be introduced. It fu*** with PEP 8 in a rather invasive way. See https://www.python.org/dev/peps/pep-0008/#id17.